### PR TITLE
Milestone 5

### PR DESCRIPTION
**Pull request by garciajordy**
# Calculator

**App.js:**
- Add Total, Next and Operator to my state.
- Add the Calculator function to my handleClick.
- Add "or" statement to my display component to display either next or total.

_I already implemented the handleClicks on the previous milestone so this one was very simple!_
_I haven't done any styling as it was not required yet, but I will do it once my project got approved!_
_Btw check out the live link in the README file!_

_Thank you for taking the time to review my project!_ 😄

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -1,30 +1,24 @@
...
-    if (value.total === '0') {
-      newValue = '';
-    } else {
-      newValue = value.total;
-    }
-
-    if (name === 'AC') {
-      setValue(value.total = { total: '0' });
-    } else {
-      newValue += name;
```
Comment (trekab) [inline src/components/App.js:20]: - [ ] Convert the `App` component to a stateful component using the __class__ syntax. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone5/app/stateful.png))
- [ ] Have the state initialized in constructor ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone5/app/stateful.png))

**Review by trekab — CHANGES_REQUESTED** [review]
#### Changes Required! :recycle: 
Nicely done @garciajordy! The functionality is well implemented but there is one requirement you seem to have missed about this milestone. As highlighted in the comments below, the `App` component is expected to be a class-based component. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone5/app/stateful.png)) :thinking: 

:bulb:  _This is aimed at giving you a chance to work with class-based components_.

__Please make the necessary changes and submit another code review request.__
_Feel free to reach me on slack for any inquiries about this review._ :wink: 
__[Treasure Kabareebe](https://trekab.github.io)__

**Comment by garciajordy** [comment]
# Changes

**App.js:**
- Change the App component to a class like requested.
- Use Constructor and super like requested.
- Use setState like requested.


_Thanks for taking the time to review my project!_ 😄

**Review by bellom — APPROVED** [review]
## Hi @garciajordy  :clap: 

**Great work making all the necessary corrections from the previous review.**

# Status: Approved 👍 

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit the project completion form.

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Pull request merged**